File: C#/Object-Oriented-Programming/BasicObjectCreation/BasicObjectCreation.cs
using Object_Oriented_Programming.BasicObjectCreationImpl;

namespace Object_Oriented_Programming
{
    class BasicObjectCreation
    {
        public static void Invoke()
        {
            Console.WriteLine("Basic Object Creation");
            Employee employee = new Employee(1, "John Doe");
            employee.Display();
        }
    }
}

File: Java/Object-Oriented-Programming/BasicObjectCreation/BasicObjectCreation.java
public class BasicObjectCreation {
    public static void main(String[] args) {

        System.out.println("Basic Object Creation in Java");
        // Create an instance of the Employee class
        Employee employee = new Employee(1, "Ramprakash");
        
        // Access and print the employee's details
        System.out.println("Employee ID: " + employee.getId());
        System.out.println("Employee Name: " + employee.getName());
        
        // Modify the employee's details
        employee.setName("Rakshit");
        
        // Print the updated employee details
        System.out.println("Updated Employee: " + employee);
    }
    
}